### DISCUSSION TITLE
win11构建示例时报错

### DISCUSSION THREAD

**Discussion by Laboor** [hathach/tinyusb#2617] [General]
按照文档所描述的语句构建项目时报错。我构建的是**tinyusb\examples\device\audio_test_multi_rate**项目，构建的语句是**make BOARD=stm32f439nucleo all**，控制台提示的错误如下：
<img width="866" alt="微信图片_20240502142715" src="https://github.com/hathach/tinyusb/assets/26828462/2494b000-ea19-4aa6-adf3-574a4bd1e3b5">
请问下这是什么原因导致的呢，我试过更换make版本，但没有作用，或者那位好心人能给我发个构建好的这个示例，，感谢万分！

**Comment by hathach** [comment]
Please use English so that everyone can understand
